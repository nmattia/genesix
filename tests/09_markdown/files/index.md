# Hello

Welcome to my website!!!

The other page:

* [other-page]
* [other-page-2]

[other-page]: ./page.md
[other-page-2]: page.md
