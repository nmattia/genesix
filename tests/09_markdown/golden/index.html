<h1 id="hello">Hello</h1>
<p>Welcome to my website!!!</p>
<p>The other page:</p>
<ul>
<li><a href="page.html">other-page</a></li>
<li><a href="page.html">other-page-2</a></li>
</ul>
